src/webob/request.py

```diff
@@ -733,7 +733,7 @@ def _json_body__get(self):
         return json.loads(self.body.decode(self.charset))
 
     def _json_body__set(self, value):
-        self.body = json.dumps(value, separators=(",", ":")).encode(self.charset)
+        self.body = json.dumps(value).encode(self.charset)
```

Comment:
@Toni is this PR also applied on the Acorn, and it's working? I expect that simplify replacing json by orjson doesn't work.  orjson.dumps returns bytes and json.dumps returns string ;)